versioned_docs/version-v1.0/settings/settings.md

```diff
@@ -78,6 +78,7 @@ The following options and values can be set:
 :::caution
 
 If you configure `httpProxy` and `httpsProxy`, you must also put Harvester node's CIDR into `noProxy`, otherwise the Harvester cluster will be broken.
+And if you also configure the `cluster-registration-url` setting, you usually need to add the host of the `cluster-registration-url` into `noProxy` as well.
```

Comment:
```suggestion
If you also configure `cluster-registration-url`, you usually need to add the host of `cluster-registration-url` to `noProxy` as well, otherwise you cannot access the Harvester cluster from Rancher.
```